aliases/available/general.aliases.bash

```diff
@@ -75,9 +80,34 @@ alias rd='rmdir'
 alias shit='bash-it'
 alias batshit='bash-it'
 alias bashit='bash-it'
+alias bsh='bash-it'
+alias batbsh='bash-it'
+alias babsh='bash-it'
 alias bash_it='bash-it'
 alias bash_ti='bash-it'
 
+# Additional bash-it aliases for help/show
+alias bshsa='bash-it show aliases'
+alias bshsc='bash-it show completions'
+alias bshsp='bash-it show plugins'
+alias bshha='bash-it help aliases'
+alias bshhc='bash-it help completions'
+alias bshhp='bash-it help plugins'
+alias bshsch="bash-it search $1"
+alias bshschen="bash-it search $1 --enable"
+alias bshenp="bash-it enable plugin"
+alias bshena="bash-it enable alias"
+alias bshenc="bash-it enable completion"
+
+# Shorten extract
+alias xt="extract"
+
+# source ~/.bash_profile
+alias sprof="source ~/.bash_profile"
```

Comment:
Yes. This shouldn't have arrived here either.  I'll be more rigorous in the future.